backend/src/main/java/gov/cdc/usds/simplereport/service/TestEventService.java

```diff
@@ -16,6 +16,6 @@ public TestEventService(TestEventRepository terepo) {
   }
 
   public TestEvent getLastTestResultsForPatient(Person patient) {
-    return _terepo.findFirst1ByPatientOrderByCreatedAtDesc(patient);
+    return _terepo.findFirstByPatient(patient);
```

Comment:
Is this query still ordered by created at date? I would prefer keeping the logic the same generally.